src/common.props

```diff
@@ -2,14 +2,14 @@
   <PropertyGroup>
     <Copyright>Copyright © 2013-2017 Akka.NET Team</Copyright>
     <Authors>Akka.NET Team</Authors>
-    <VersionPrefix>1.3.1</VersionPrefix>
+    <VersionPrefix>1.3.2</VersionPrefix>
```

Comment:
Bumped a version number

Reply:
@Aaronontheweb I think we should re-run all tests if `common.props` was changed. But our incremental testing did not do it

Reply:
@alexvaluyskiy that's a good call - I'll look into adding it